crates/glaredb/src/server.rs

```diff
@@ -57,14 +58,15 @@ pub struct ComputeServerBuilder {
     disable_rpc_auth: bool,
     enable_simple_query_rpc: bool,
     enable_flight_api: bool,
+    metastore_bucket: Option<String>,
+    metastore_local_file_path: Option<PathBuf>,
```

Comment:
so this already took a `service_account_path`, and just to double-check that I have it right, we are adding some capabilities to this SA that was previously on a separate metastore SA?

That tracks to me, just wanted to double check.

We should make that change and deploy this branch to QA to test before merging it into main.

We can also phase out the metastore key until v0.10.0 or later releases

Reply:
the local file path is "how do we run in single node with the metastore in local files" rather than remote in a service (current cloud implementation) or in memory (tests/etc.)

This option is nil/none (I think) on cloud, and I think (re another comment) I can get rid of it to just be a sub-directory of "data-dir"... 

I think I will remove this option entirely and put it in

Reply:
This comment is not well placed, the line of code it's referring to is:

https://github.com/GlareDB/glaredb/blob/6a03e0ea84353dbd9c1118f085d515f46c9bb393/crates/glaredb/src/server.rs#L52

Reply:
I believe that's correct. 

We'll be able to test this out on QA (or from here, if you want?) and I'm down to find a time when we can pair on this.

Reply:
Yea, all we need to do is comment out this line:

https://github.com/GlareDB/glaredb/blob/6a03e0ea84353dbd9c1118f085d515f46c9bb393/.github/workflows/ci.yaml#L623

and then I can roll it out to QA, along with dropping metastore.

We should look to see it fail first and then merge the SA perms and verify it works.

We can pair on this tomorrow (Friday) if you'd like, it would be a good task for me after the release stuff.